app/components/HeaderComp.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import AddToCart from "./AddToCart"
+import { HEADER_LABEL_MESSAGE } from '../constant';
+import logo from '../images/logo.png'
+import cart from '../images/cart.svg'
+
+const HeaderComp = (props) => {
+    const { cartItem, handleCount } = props;
+    const [toggleCart, settoggleCart] = useState(false)
+    const openAddtocart = (flag) => {
+        settoggleCart(flag)
+    }
+    return (
+        <header className="header">
+            <div className="inner-header floatcontainer">
+                <div className="logo-container">
+                    <img src={logo} alt="App Logo" />
+                </div>
+                <ul className="navigation">
+                    <li> <NavLink exact to="/" role="button" className="signInBar" activeClassName="selected" >{HEADER_LABEL_MESSAGE.HEADER_LINK1}</NavLink></li>
+                    <li> <NavLink to="/product" role="button" className="signInBar" activeClassName="selected" >{HEADER_LABEL_MESSAGE.HEADER_LINK2}</NavLink></li>
+                </ul>
+                <div className="navigation-right">
+                    <NavLink to="/login" role="button" className="signInBar" activeClassName="selected" >{HEADER_LABEL_MESSAGE.HEADER_LINK3}</NavLink>
+                    <NavLink to="/ragister" role="button" className="registerBar" activeClassName="selected" >{HEADER_LABEL_MESSAGE.HEADER_LINK4}</NavLink>
+                    <div className="cart-container" >
```

Comment:
what is the rationale behind giving a role of button to links?

Reply:
removed, and add role navigation in parent div.